packages/web-runtime/src/pages/account.vue

```diff
@@ -28,51 +24,51 @@
           <translate>Edit</translate>
         </oc-button>
       </div>
-      <hr />
-      <h2 v-translate class="oc-text-bold oc-text-initial oc-mb">Account Information</h2>
-
-      <dl class="account-page-info oc-flex oc-flex-wrap">
-        <div class="account-page-info-username oc-mb oc-width-1-2@s">
-          <dt v-translate class="oc-text-normal oc-text-muted">Username</dt>
-          <dd>
-            {{ user.username || user.id }}
-          </dd>
-        </div>
-        <div v-if="user.username && user.id" class="account-page-info-userid">
-          <dt v-translate class="oc-text-normal oc-text-muted">User ID</dt>
-          <dd>
-            {{ user.id }}
-          </dd>
-        </div>
-        <div class="account-page-info-displayname oc-mb oc-width-1-2@s">
-          <dt v-translate class="oc-text-normal oc-text-muted">Display name</dt>
-          <dd>
-            {{ user.displayname }}
-          </dd>
-        </div>
-        <div class="account-page-info-email oc-mb oc-width-1-2@s">
-          <dt v-translate class="oc-text-normal oc-text-muted">Email</dt>
-          <dd>
-            <template v-if="user.email">{{ user.email }}</template>
-            <span v-else v-translate>No email has been set up</span>
-          </dd>
-        </div>
-        <div class="account-page-info-groups oc-mb oc-width-1-2@s">
-          <dt
-            v-translate
-            class="oc-text-normal oc-text-muted"
-            @click="$_oc_settingsAccount_getGroup"
+    </div>
+    <h2 v-translate class="oc-text-bold oc-text-initial oc-mb">Account Information</h2>
+    <dl class="account-page-info oc-flex oc-flex-wrap">
```

Comment:
mostly indendation changes from here on... only line 59 and following have actually changed.